scripts/build/entries/index.ts

```diff
@@ -0,0 +1,257 @@
+import { exec } from 'node:child_process';
+import { join } from 'node:path';
+
+import type { BuildEntriesByPackageName } from '../utils/entry-utils';
+
+export const buildEntries = {
```

Comment:
Could you add some JSDoc comments or a readme for other developers to elaborate on what to do if I want to build a new entry and what to consider?